neuralop/datasets/data_transforms.py

```diff
@@ -45,6 +45,11 @@ def wrap(self, model):
     def forward(self, x):
         pass
 
+    def train(self):
```

Comment:
I don't see the need for this class. Not only does it not bring feature it actually breaks the default behaviour of the functions (it should at the very least call super). I'd be in favor of removing it alltogether.

Reply:
update